frontend/uddhar-frontend/src/organization/data/data.jsx

```
@@ -0,0 +1,41 @@
+        return registeredId.includes(disaster.disaster_id);
+    });
+    return registeredDisaster.map((disaster) => {
+        return {
+            disaster_id: disaster.disaster_id,
+            title: disaster.title,
+            description: disaster.description,
+            location: disaster.location,
+            startDate: disaster.startDate,
+            status: disaster.status,
+            type: disaster.type,
+            image: eval(disaster.type.toLowerCase()),
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)